modules/ROOT/pages/connected-app-bearer-token-example.adoc

```diff
@@ -0,0 +1,58 @@
+= Getting the Bearer Token for a Connected App Example
+
+This example demonstrates how to create a connected app that acts on its own behalf using client credentials to get the bearer token so you can invoke Anypoint Platform APIs. 
+
+[[create-connected-app]]
+== Create a Connected App
+
+To create a connected app, add Runtime Manager scopes, and deploy the app to CloudHub:
+
+include::partial$include-nav-connected-apps.adoc[]
+
+. In the *Owned Apps* section, click *Create App*.
+. In the *Create App* page, enter a unique *Name* for the app, select *App acts on its own behalf (client credentials)*, and then click *Save*.
+. Click *Add Scopes*.
+. Select the scopes for the app from the *Runtime Manager* section, and then click *Next*. 
++
+For information about scopes and permissions for Connected Apps, see the xref:connected-apps-developers.adoc#endpoint-scopes[Endpoint Scopes].
```

Comment:
```suggestion
For information about scopes and permissions for connected apps, see the xref:connected-apps-developers.adoc#endpoint-scopes[Endpoint Scopes].
```

Reply:
https://confluence.internal.salesforce.com/display/MTDT/Term+Presentation+Reference#TermPresentationReference-C:~:text=connected%20apps%2C%20the,the%20proper%20noun.